### ISSUE TITLE
Details of training syncnet

### ISSUE THREAD

**Issue by sunwoo76** [guanjz20/StyleSync_PyTorch#2]
Hello :)

I have a question about the training on your own syncnet.

As far as I know, it is difficult to converge the losses of Syncnet. So, many users have left messages in the Issues tab of the Wav2Lip GitHub repository.

Could you explain the architecture or training scheme of Syncnet of this model in more detail?

Thanks! :_)

**Comment by saeedfirouzi** [comment]
why dont you just use syncnet in wa2lip and its pretrain file?

**Comment by spark-hudson** [comment]
I am also interested in knowing how the syncnet is trained. Is eq (4) in the paper the same loss that was used during the training of syncnet?

Thanks in advance.

**Comment by madgnot** [comment]
zhou的工作总是有问题的哦 水论文有一手

**Comment by PolarRobin** [comment]
Would love get some details as well :)
